addons/api/tests/unit/models/worker-test.js

```diff
@@ -71,6 +71,38 @@ module('Unit | Model | worker', function (hooks) {
     const model = store.peekRecord('worker', workerId);
     await model.setApiTags(tags);
   });
+  
+  test('it has a `removeApiTags` method that targets a specific POST API', async function (assert) {
```

Comment:
I am a bit confuse by this test, is a unit test but we are mocking quite a bit of stuff and it has no assertions? I am sure I am missing context, but can you ellaborate a bit the reasoning for this test?

Reply:
The assertion is on line 85. We are mocking it here so we can assert the correct payload is added to the `post` request. Does this help explain what is going on here?

Reply:
@cameronperera 

For the sake of this PR. I will be ok as it is, there are previous tests within this same file following same approach. But definitely I think we should bring this topic to discussion in tech time.